wallet/concepts/multichain-api.md

```diff
@@ -0,0 +1,116 @@
+---
+description: Learn about the Multichain API.
+sidebar_custom_props:
+  flask_only: true
+---
+
+# About the Multichain API
+
+:::flaskOnly
+:::
+
+The Multichain API is a scalable, generalized web3 wallet API that supports simultaneous
+interactions across multiple blockchain networks.
```

Comment:
the terms network and ecosystems are overloaded, but is it clearer to mention `networks and ecosystems`? I take networks to mean chains inside an ecosystem

Reply:
There was a thread about "network" terminology in Slack [here](https://consensys.slack.com/archives/C03Q3TCUVRT/p1743528144279929). I think the consensus was to continue to use "networks" for end users, but developers have more bandwidth to understand EVM vs. non-EVM. I think using "networks and ecosystems" here makes sense, added :+1: